### PR TITLE
chore: update journey schema

### PR DESCRIPTION
**Pull request by peschina**
Update journey schema

### CODE REVIEW

File: src/client/clients/journey/schema.ts [hunk 1]
```diff
@@ -45,6 +45,9 @@ export interface paths {
                     offset?: number;
                     limit?: number;
                     ids?: string | string[];
+                    search?: string;
+                    from?: string;
```
Comment (Thurfjell) [inline src/client/clients/journey/schema.ts:49]: from what?

Comment (peschina) [reply src/client/clients/journey/schema.ts:49]: `from` and `to` are used to filter by time range

Comment (Thurfjell) [reply src/client/clients/journey/schema.ts:49]: Would it be breaking any standards to accentuate that with eg. `fromAt` or `fromDate`? Not a deal breaker of course.

Comment (peschina) [reply src/client/clients/journey/schema.ts:49]: I just used the same naming we have in other api filters, but happy to change it in case

Comment (Thurfjell) [reply src/client/clients/journey/schema.ts:49]: oki

**Review by Thurfjell — APPROVED** [review]

**Review by alewin — APPROVED** [review]

**Pull request merged**